doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1093,6 +1093,8 @@ The Dataverse Software S3 driver supports multi-part upload for large files (ove
 First: Set Up Accounts and Access Credentials
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+**Note:** As of version 5.14, if Dataverse is running in an EC2 instance it will prefer RBAC for the S3 default profile, even if administrators configure Dataverse with programmatic access keys. Named profiles can still be used to override RBAC for specific datastores. This is preferential from a security perspective as there are no keys to rotate or have stolen. If you intend to assign a role to your EC2 instance, you will still need the ``~/.aws/config`` file to specify the region but you need not generate credentials for the default profile. For more information please see https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2.html
```

Comment:
Sorry - there was more changed than just the part in parens (which I was suggesting to keep). In line 1 I changed 'for the S3 default profile' to say 'over' instead of 'for', I spelled out RBAC the first time as Role Based Access Control, and in the third sentence I changed 'This' to 'RBAC' Since the sentence about named profiles made 'This' ambiguous.